main.go

```diff
@@ -222,3 +234,54 @@ func getCloudFoundryRedisBindings() (addrs, passwords, aliases []string) {
 
 	return
 }
+
+// loadRedisSentinel gets Redis IP addresses and Ports and return proper information.
+func loadRedisSentinel(addr, password, alias, separator string) ([]string, []string, []string, error) {
+	passwords = strings.Split(password, separator)
+	aliases = strings.Split(alias, separator)
+
+	options := []redis.DialOption{
+		redis.DialConnectTimeout(5 * time.Second),
+		redis.DialReadTimeout(5 * time.Second),
+		redis.DialWriteTimeout(5 * time.Second),
+	}
+
+	sentinel_addrs := strings.Split(addr, separator)
+	sntnl := &sentinel.Sentinel{Addrs: sentinel_addrs, MasterName: *masterName,
+		Dial: func(addr string) (redis.Conn, error) {
+			c, err := redis.Dial("tcp", addr, options...)
+			if err != nil {
+				return nil, err
+			}
+			return c, nil
+		},
+	}
+
+	var addrs []string
+	master, err := sntnl.MasterAddr()
+	if err != nil {
+		log.Warnln("Error while getting address of redis master", err)
+		return nil, nil, nil, err
+	} else {
+		addrs = append(addrs, master)
+
+		slaves, err := sntnl.Slaves()
+		if err != nil {
```

Comment:
This will make the exporter exit due to the `Fatalf` in line 90 - is that intended and desired? Is there a scenario where it makes to continue with just he master only if retrieving the slaves in line 268 failed?